ui/app/AppLayouts/Chat/views/ContactsColumnView.qml

```diff
@@ -431,52 +430,46 @@ Item {
         }
     }
 
-    // Not Refactored Yet
-//    Connections {
-//        target: root.store.chatsModelInst.communities
-//        onImportingCommunityStateChanged: {
-//            if (state !== Constants.communityImported &&
-//                state !== Constants.communityImportingInProgress &&
-//                state !== Constants.communityImportingError)
-//            {
-//                return
-//            }
-
-//            if (state === Constants.communityImported)
-//            {
-//                if (Global.toastMessage.uuid !== communityImportingProcessId)
-//                    return
-
-//                Global.toastMessage.close()
-
-//                //% "Community imported"
-//                Global.toastMessage.title = qsTrId("community-imported")
-//                Global.toastMessage.source = ""
-//                Global.toastMessage.iconRotates = false
-//                Global.toastMessage.dissapearInMs = 4000
-//            }
-//            else if (state === Constants.communityImportingInProgress)
-//            {
-//                Global.toastMessage.uuid = communityImportingProcessId
-//                //% "Importing community is in progress"
-//                Global.toastMessage.title = qsTrId("importing-community-is-in-progress")
-//                Global.toastMessage.source = Style.svg("loading")
-//                Global.toastMessage.iconRotates = true
-//                Global.toastMessage.dissapearInMs = -1
-//            }
-//            else if (state === Constants.communityImportingError)
-//            {
-//                if (Global.toastMessage.uuid !== communityImportingProcessId)
-//                    return
-
-//                Global.toastMessage.close()
-//                return
-//            }
-
-//            Global.toastMessage.displayCloseButton = false
-//            Global.toastMessage.displayLink = false
-//            Global.toastMessage.iconColor = Style.current.primary
-//            Global.toastMessage.open()
-//        }
-//    }
+    Connections {
+        target: root.store.communitiesModuleInst
```

Comment:
Seems to be not aligned with our QML architecture. Not sure how to solve such a case in a different way though..

Reply:
@saledjenic this is not the case here. We already have a level of indirection in the store:
```
property var communitiesModuleInst: communitiesModule
```
so it would be "wrong" to use here `communitiesModule` directly, but using `communitiesModuleInst` is totally fine.